phac-nml/irida-next
# Members pagination

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR adds in pagination to the members table for groups and projects.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

![image](https://github.com/phac-nml/irida-next/assets/25466211/73e71421-8e50-4fcb-9901-dcd30f481728)

![image](https://github.com/phac-nml/irida-next/assets/25466211/1c413ef0-6abd-4be5-a05a-cccdd361643b)

![image](https://github.com/phac-nml/irida-next/assets/25466211/f3887cf3-2bf9-4b4a-90c1-b543e138641a)


## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Login and visit any project or group members page
2. You should see pagination information displayed
3. Try adding 20+ users to either a group or project
4. Verify that you can now see the Prev and Next buttons for pagination and clicking on another page in the pagination works

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
